### ISSUE TITLE
[New feature] Plotting / Vizualization

### ISSUE THREAD

**Issue by lrnv** [lrnv/Copulas.jl#152]
AnderGray suggested in the Joss review to add functionality for Plotting / Visualisation.

---------

Yes, this is indeed a very good idea. We might borow a few plots from `BivariateCopulas.jl` and also revamp #78, but my goal is to be compatible with Plots.jl and StatsPlots.jl, and do that as a package extension so that loading these things is not necessary to use the package. It will also allow to add a lot of graphs to the documenttion.

**Comment by alecloudenback** [comment]
You might just point folks to https://sefffal.github.io/PairPlots.jl/dev/guide/

**Comment by lrnv** [comment]
@alecloudenback Yes this is the kind of stuff I want in the package extension, the ideal plot here would be: 

- bottom triangle plots bivariate contours on the rank scale [0,1]^2
- upper triangle plots bivariate contours on the z-scale (standard Gaussian marginals)
- (for `plot(::SklarDist)`) The diagonal represents the marginal densities with a `plot(::UnivariateDistribution)`, e.g. the one from StatsPlots. 
- (for `plot(::Copula)`) Diagonal elements are TBD, maybe simply variable names ? or something else ? 

If PairsPlots.jl can be used here, then perfect.

**Comment by lrnv** [comment]
@AnderGray I have finally ported some of your plots from BivariateCopulas.jl into this package, you can view someof them there: https://lrnv.github.io/Copulas.jl/dev/visualizations/. I hope you do not mind us using your code ? Tell me what you think.

**Comment by AnderGray** [comment]
@lrnv very happy to take a look, and absolutely you can use the code! I can't seem to open the link however

**Comment by lrnv** [comment]
@AnderGray this one should work: https://lrnv.github.io/Copulas.jl/dev/manual/visualizations
